### ISSUE TITLE
Art title origins as annotations on titles

### ISSUE THREAD

**Issue by rjyounes** [LD4P/ArtFrame-RareMat#23]
Should this be changed to the bf:Note model, since we are making these updates elsewhere in the models?

**Comment by melanieWacker** [comment]
Absolutely. It's on my to-do list. We are also reviewing the proposed subclasses, so I will make all the changes at the same time.
